lib/puppet/type/java_ks.rb

```diff
@@ -152,6 +152,13 @@ def self.title_patterns
         ]
       ],
       [
+        /^(.*):([a-z]:(\/|\\).*)$/i,
```

Comment:
wouldn't it be enough to have `/^(.)/` or `(.?)` if the letter is really optional?